local-cli/generator-android/templates/src/app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +13,7 @@
       <activity
         android:name=".MainActivity"
         android:label="@string/app_name"
+		android:screenOrientation="portrait"
```

Comment:
@SandroMachado Yeah, but it works fine in landscape, so no reason we should lock Android to portrait only because iOS does it. Maybe we should change iOS generator to not lock it into portrait mode.